apps/crn-server/scripts/export-entity.ts

```diff
@@ -15,8 +15,8 @@ import Events from '../src/controllers/events';
 import ExternalAuthors from '../src/controllers/external-authors';
 import ResearchOutputs from '../src/controllers/research-outputs';
 import Users from '../src/controllers/users';
-import AssetDataProvider from '../src/data-providers/assets.data-provider';
-import UserDataProvider from '../src/data-providers/users.data-provider';
+import { AssetSquidexDataProvider } from '../src/data-providers/assets.data-provider';
+import { UserSquidexDataProvider } from '../src/data-providers/users.data-provider';
```

Comment:
I can understand this change, but this is an interface. I would have thought the interface would be generic whilst the implementation would be Squidex specific.

Reply:
it's not though, this is the implentation, the interface remains generic:
https://github.com/yldio/asap-hub/blob/a5636133d05264002fa6b080cfedd43d727b9fdd/apps/crn-server/src/data-providers/assets.data-provider.ts#L6